master/reference/public-cloud/aws.md

```diff
@@ -0,0 +1,113 @@
+---
+title: AWS
+---
+
+Calico is designed to provide high performance massively scalable virtual
+networking for containers in public cloud environments such as
+Amazon Web Services (AWS).
+
+## Why Use Calico in AWS
+
+#### Network Policy for Containers
+
+While AWS Security Groups control network traffic sent to ec2 instances, it
+does not cover containers running on hosts. Calico automatically implements
+fine-grain, dynamic network policy for containers.
+
+#### No Overlays
+
+Calico minimizes the need for overlays by assigning each container
+its own IP and allowing containers to talk to one another in a pure IP network.
+
+#### No 50 Node Limit
+
+[AWS VPC Routing tables are limited to 50 programmable routes](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Appendix_Limits.html#vpc-limits-route-tables).
+While this might limit other networking plugins, Calico supports
+ip-over-ip encapsulation to circumvent manually programming container routes in
+the VPC routing table altogether.
+
+## How to Run Calico in AWS
+
+To deploy Calico in AWS, you must ensure that the proper security group rules
+have been made and that traffic between containers on different hosts is not
+dropped by the VPC. There are a few different options for doing this depending
+on your deployment.
+
+#### Configure Security Groups for BGP
```

Comment:
It's not just BGP that needs security group allowances.  You should also mention that when using `ipip` you need to allow IP protocol 4 in the security group or else `ipip` traffic will be dropped.